WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasDisplayButton.lua

```diff
@@ -937,7 +937,7 @@ local methods = {
                     self.frame:SetScript("OnClick", nil)
                 end
             end
-        else
+        elseif (self.dragging) then
```

Comment:
while it is only called in `HideOptions()`, this branch needs to enable and restore `OnClick` on _all_ displays not only on the dragged elements the `Drop()` can be restricted to only be called when dragged.